src/statemap.rs

```diff
@@ -1401,13 +1381,12 @@ impl Statemap {
             for entity in data.keys() {
                 println!("{}{}: [", comma, entity);
 
-                let datum = data.get(entity).unwrap();
+                let datum = &data[entity];
 
-                for i in 0..datum.len() - 1 {
+                for i in 0..datum.len() {
                     println!("{},", datum[i]);
                 }
 
-                println!("{}", datum[datum.len() - 1]);
```

Comment:
I may have misunderstood this bit entirely, but I really don't understand why it was not in the loop. It doesn't seem to be any different.